cookbooks/wiki/templates/default/LocalSettings.php.erb

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
Either this template is needed, in which some recipe needs to instantiate it, or it's not in which case it should be removed.